module/sheets/actor-sheet.mjs

```diff
@@ -340,6 +343,74 @@ export class BNBActorSheet extends ActorSheet {
     context.hps = context.system.attributes.hps;
   }
 
+  async _prepareEnrichedFields(context, actorType) {
```

Comment:
attempting to organize the enriched data the same way as the actual template data.